src/hooks/useMember/utils.ts

```diff
@@ -0,0 +1,26 @@
+import { promisify } from '@/utils/data'
+import { readFromLocalStorage, writeToLocalStorage } from '@/utils/localStorage'
+import { Member } from './useMember'
+
+export const getMember = promisify(() => readFromLocalStorage<Member>('member') || null)
+
+export const setMember = async (member: Member) => {
+  writeToLocalStorage('member', member)
+  const newMember = await getMember()
+  return newMember
+}
+
+export const setActiveChannel = async (channelId: string) => {
+  const member = await getMember()
+  if (!member?.id) {
```

Comment:
```suggestion
  if (!member) {
```
Will also work, right?